src/configurations.ts

```diff
@@ -151,6 +151,10 @@ const configurations = {
         url: readStringValue('RENDERER_URL', 'http://localhost:3000'),
         requestTimeout: readIntValue('RENDERER_REQUEST_TIMEOUT', 75000),
     },
+    openlab: {
+        url: readStringValue('OPENLAB_URL', 'https://openlab.citytech.cuny.edu/ol-webwork/'),
```

Comment:
we probably don't want to publicly expose the openlab url, this one might be better to leave empty string

The only other one we default is jira which i'm less worried about since it is jira cloud and I like the example

Reply:
stripped